interactive_engine/src/executor/store/Cargo.toml

```diff
@@ -26,6 +26,7 @@ maxgraph-common = { path = "../../common/rust/common" }
 
 [dependencies.rocksdb]
 git = "https://github.com/pingcap/rust-rocksdb.git"
+rev = "23a604459899f4f7a4c8967f16b6fa623474f27b"
```

Comment:
Why relies on some `rev` here?

Reply:
This is because the new commits of rust-rocksdb have introduced an incompatible function we used, which leads to compile failure. So I add the  commit id from the Cargo.lock for compatible (with older version) consideration.

Reply:
This is because the new commits of rust-rocksdb have introduced an incompatible function we used, which leads to compile failure. So I add the  commit id from the Cargo.lock for compatible (with older version) consideration.

Reply:
This is very odd. Instead, fork a github version to GraphScope to stay in whatever version we'd like is a more common practice.